Azure/azure-cli
# [RDBMS] Add single server show-connection-string and tests for local-context commands, server creation

**Description**  

- az mysql server show-connection-string
- az postgres server show-connection-string
- test for local-context flexible server mgmt
- added test to flexible-server mgmt

**Testing Guide**  

- azdev test ServerMgmtScenarioTest
- azdev test FlexibleServerMgmtScenarioTest
- azdev test FlexibleServerLocalContextScenarioTest



---

This checklist is used to make sure that common guidelines for a pull request are followed.
